### PR TITLE
chore: many trivial spelling and layout fixes

### PR DESCRIPTION
**Pull request by ckielstra**
Fixed spelling
Removed double spaces
Added space after ','

### CODE REVIEW

File: scripts/style_api_gen.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
  'style_type': 'num',   'var_type': 'lv_coord_t',  'default':0, 'inherited': 0, 'layout': 1, 'ext_draw': 1,
  'dsc': " Rotate image-like objects. Added to the rotation set on the object. The value is interpreted in 0.1 degree unit. E.g. 45 deg. = 450 " },
 
-{'section': 'Padding', 'dsc':'Properties to describe spacing betwwen the parent\'s sides and the children and among the children. Very similar to the padding properties in HTML.' },
+{'section': 'Padding', 'dsc':'Properties to describe spacing betewen the parent\'s sides and the children and among the children. Very similar to the padding properties in HTML.' },
```
Comment (kisvegabor) [inline scripts/style_api_gen.py]:
```suggestion
{'section': 'Padding', 'dsc':'Properties to describe spacing between the parent\'s sides and the children and among the children. Very similar to the padding properties in HTML.' },
```

File: scripts/style_api_gen.py [hunk 2]
```diff
@@ -161,19 +161,19 @@
 
 {'name': 'BORDER_OPA',
  'style_type': 'num',   'var_type': 'lv_opa_t' ,  'default':'`LV_OPA_COVER`', 'inherited': 0, 'layout': 0, 'ext_draw': 0,
- 'dsc': "Set the opcitiy of the border. Value 0, `LV_OPA_0` or `LV_OPA_TRANSP` means fully transparent, 256, `LV_OPA_100` or `LV_OPA_COVER` means fully covering, other values or LV_OPA_10, LV_OPA_20, etc means semi transparency."},
+ 'dsc': "Set the opacitiy of the border. Value 0, `LV_OPA_0` or `LV_OPA_TRANSP` means fully transparent, 256, `LV_OPA_100` or `LV_OPA_COVER` means fully covering, other values or LV_OPA_10, LV_OPA_20, etc means semi transparency."},
```
Comment (kisvegabor) [inline scripts/style_api_gen.py]:
```suggestion
 'dsc': "Set the opacity of the border. Value 0, `LV_OPA_0` or `LV_OPA_TRANSP` means fully transparent, 256, `LV_OPA_100` or `LV_OPA_COVER` means fully covering, other values or LV_OPA_10, LV_OPA_20, etc means semi transparency."},
```

File: src/draw/sw/lv_draw_sw.h [hunk 3]
```diff
@@ -51,9 +51,8 @@ void lv_draw_sw_arc(lv_draw_ctx_t * draw_ctx, const lv_draw_arc_dsc_t * dsc, con
 void lv_draw_sw_rect(lv_draw_ctx_t * draw_ctx, const lv_draw_rect_dsc_t * dsc, const lv_area_t * coords);
 
 void lv_draw_sw_bg(lv_draw_ctx_t * draw_ctx, const lv_draw_rect_dsc_t * dsc, const lv_area_t * coords);
-
-void lv_draw_sw_letter(lv_draw_ctx_t * draw_ctx, const lv_draw_label_dsc_t * dsc,  const lv_point_t * pos_p,
-                       uint32_t letter);
+  
+void lv_draw_sw_letter(lv_draw_ctx_t * draw_ctx, const lv_draw_label_dsc_t * dsc, const lv_point_t * pos_p,
```
Comment (kisvegabor) [inline src/draw/sw/lv_draw_sw.h:54]:
```suggestion
void lv_draw_sw_letter(lv_draw_ctx_t * draw_ctx, const lv_draw_label_dsc_t * dsc, const lv_point_t * pos_p,
                       uint32_t letter);
```

**Comment by kisvegabor** [comment]
Woooow, thank you very much! Now LVGL is really professional! :slightly_smiling_face: 

The CI fails because `lv_conf_template.h` was changed but `scripts/lv_conf_internal_gen.py` wasn't run to update `src/lv_conf_internal.h`.

So could you run `lv_conf_internal_gen.py`. If not for any reason (e.g. no Python installed) I can do it too.

File: src/draw/sw/lv_draw_sw.h [hunk 4]
```diff
@@ -51,8 +51,8 @@ void lv_draw_sw_arc(lv_draw_ctx_t * draw_ctx, const lv_draw_arc_dsc_t * dsc, con
 void lv_draw_sw_rect(lv_draw_ctx_t * draw_ctx, const lv_draw_rect_dsc_t * dsc, const lv_area_t * coords);
 
 void lv_draw_sw_bg(lv_draw_ctx_t * draw_ctx, const lv_draw_rect_dsc_t * dsc, const lv_area_t * coords);
-
-void lv_draw_sw_letter(lv_draw_ctx_t * draw_ctx, const lv_draw_label_dsc_t * dsc,  const lv_point_t * pos_p,
+
```
Comment (kisvegabor) [inline src/draw/sw/lv_draw_sw.h]:
```suggestion

```

**Comment by ckielstra** [comment]
I didn't know about the script for updating lv_conf_internal.h. That should be fixed now.
I see you also fixed 2 more spelling issues. :-)

**Comment by ckielstra** [comment]
Hmm, still an error on lv_conf_internal.h? 
@kisvegabor Can you have a look at this? I get no more differences when I run `scripts/lv_conf_internal_gen.py`

**Comment by embeddedt** [comment]
I think the MicroPython errors are fixed already in `master`, so this is ready to be merged. Thanks!

**Pull request merged**